docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -35,6 +35,87 @@ The system repeatedly shows a warning message about overriding an inherited asse
 
 This warning, appropriate for the first assessment, incorrectly reappears on subsequent reassessments, suggesting that the application is still inheriting its assessment, even after it has been overridden. link:https://issues.redhat.com/browse/MTA-1825[MTA-1825]
 
+.Custom rule for javax.* package import is not triggered when uploaded as an XML file
+
+In {ProductShortName} 7.0.0, some XML custom rule files are converted to the new YAML format during analysis. The root cause of this issue is that Java search patterns, which use `IMPORT` or `PACKAGE` as `location`, should not end in `.{*}` but just `{*}`. 
+
+To resolve this issue, whenever a custom rule has a pattern ending in `.{*}` and location one of `IMPORT` or `PACKAGE`, the pattern `.{*}` can be changed to `{*}`.
+
+In the following example XML rule, note that the `when` condition is `javaclass references` ="javax.{*}"`. 
+
+If you amend this to `javaclass references="javax{*}"`, the rule converts correctly.
+
+
+.To view the javax-package-custom-target.windup.xml, click here.
+[%collapsible%closed]
+====
+[source,xml]
+----
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="javax-package"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset evaluates whether a custom target can be used within a custom rule
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <targetTechnology id="phil" versionRange="[7,8)" />
+    </metadata>
+    <rules>
+        <rule id="javax-package-custom-target-00001">
+            <when>
+                <javaclass references="javax{*}">
+                    <location>PACKAGE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="CUSTOM RULE for javax.* package import" effort="1" category-id="potential">
+                    <message>`javax.*` packages must be renamed to `jakarta.*` for Jakarta EE9 compatibility.</message>
+                    <link title="Renamed Packages" href="https://github.com/wildfly-extras/batavia/blob/master/impl/ecl/src/main/resources/org/wildfly/extras/transformer/eclipse/jakarta-renames.properties"/>
+                </hint>
+            </perform>
+        </rule>
+    </rules>
+</ruleset>
+----
+====
+
+The suggested workaround is to use the following YAML file instead. The YAML file is the result of a conversion from XML. 
```

Comment:
@pranavgaikwad && @mguetta1 - have implemented changes suggested.

However, Pranav's changes look like the render the second codeblock redundant

Am I ok to delete?

Reply:
Yes, no need the yaml file example